jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/vo/PermissionNotificationRequest.java

```diff
@@ -0,0 +1,52 @@
+package com.tmobile.cso.pacman.inventory.vo;
+
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Instead of creating same file two times, can you please move it to batch-commons ?

Reply:
moved